.github/actions/setup-github-action-host/action.yml

```diff
@@ -0,0 +1,69 @@
+name: Setup GitHub Action Host Workflow
+description: "A composite GitHub action used to download the Chime JS SDK repo, install dependencies and setup SauceLabs Connect."
+inputs:
+  sauce-username:
+    required: true
+    description: "The username of the SauceLabs account"
+    default: ""
+  sauce-access-key:
+    required: true
+    description: "The access key of the SauceLabs account"
+    default: ""
+  sauce-rest-url:
+    required: false
+    description: "The datacenter endpoint for the SauceLabs Connect Proxy"
+    default: "https://saucelabs.com/rest/v1"
```

Comment:
Is this needed because this action will be called by different integration test which are running on different Sauce Lab Centers? e.g., Linux running on US-West, Canary running on US-East

Reply:
So, this variable has a default url which is applicable for session in US-West region and for real device sessions. For linux canaries we basically override this url, so it is definitely needed.

Reply:
@xuesichao I have added more documentation on this parameter and the corresponding URL in the README.

Reply:
Thanks for your explanation! Nice job!